### PR TITLE
Changes in section "Python OS"

### PR DESCRIPTION
**Pull request by ksharshveer**
Added few answers to questions. Also, combined two questions into one, because one was almost a subset of another. (At least based on my answer)

I am new to contributing and stuff. Please let me know if I do something wrong, etc. Thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6297,18 +6297,51 @@ with open('file.txt', 'w') as file:
 
 <details>
 <summary>How to print current working directory?</summary><br><b>
+
+    import os
+
+    current_working_directory = os.getcwd()
```
Comment (bregman-arie) [inline README.md:6303]: print(os.getcwd())

File: README.md [hunk 2]
```diff
@@ -6297,18 +6297,51 @@ with open('file.txt', 'w') as file:
...
-<summary>Given the path <code>/dir1/dir2/file1</code> print the name of the directory where the file resides (dir2)</summary><br><b>
+    import os
+
+    filename = os.path.basename('/dir1/dir2/file1')
+
+    # Another way
+    head, tail = os.path.split('/dir1/dir2/file1')
+    
+    # tail is everything after the final slash
+    filename = tail
```
Comment (bregman-arie) [inline README.md:6319]: print(tail)

**Review by bregman-arie — CHANGES_REQUESTED** [review]
Thank you for submitting this PR. Can you make some minor changes and basically wherever possible, just print immediately the result instead of setting it in a variable first?

File: README.md (hunk 2 above)
Comment (ksharshveer) [reply README.md:6319]: Yes, sure. Just to confirm - I should `print(result)` sooner when possible, and not worry about the output besides the result like
```
print(tail)
>>> file1
```

Comment (bregman-arie) [reply README.md:6319]: Yes. You create a new variable when you do something like: `filename = tail` and it is not necessary.
You can even do something like this: print(os.path.split('/dir1/dir2/file1')[1])

Comment (ksharshveer) [reply README.md]: Right. My only concern was readability of code. I suppose the audience is familiar with the language already, and just need confirmation answers.

Anyways, I made the changes and commited. Not sure if I should create a pull request, or click the resolve conversation. For now I'll wait for your response. Let me know if everything looks ok. Thank you.

Comment (bregman-arie) [reply README.md]: Looks good. Appreciate your effort. Merging

**Pull request merged**